src/routes/at-resources.routes.js

```diff
@@ -13,7 +13,8 @@ const { dashboard,
         renderAddSprintForm,
         addSprint,
         renderEditSprintForm,
-        updateSprint} = require("../controllers/at-resources.controller");
+        updateSprint,
+        renderBacklog,} = require("../controllers/at-resources.controller");
```

Comment:
shpould be `renderBacklog} = require("../controllers/at-resources.controller");`